qtawesome/__init__.py

```diff
@@ -60,28 +111,22 @@ def _instance():
         _resource['iconic'] = None
 
     if _resource['iconic'] is None:
-        _resource['iconic'] = IconicFont(
-            ('fa',
-             'fontawesome4.7-webfont.ttf',
-             'fontawesome4.7-webfont-charmap.json'),
-            ('fa5',
-             'fontawesome5-regular-webfont.ttf',
-             'fontawesome5-regular-webfont-charmap.json'),
-            ('fa5s',
-             'fontawesome5-solid-webfont.ttf',
-             'fontawesome5-solid-webfont-charmap.json'),
-            ('fa5b',
-             'fontawesome5-brands-webfont.ttf',
-             'fontawesome5-brands-webfont-charmap.json'),
-            ('ei', 'elusiveicons-webfont.ttf', 'elusiveicons-webfont-charmap.json'),
-            ('mdi', 'materialdesignicons5-webfont.ttf',
-             'materialdesignicons5-webfont-charmap.json'),
-            ('mdi6', 'materialdesignicons6-webfont.ttf',
-             'materialdesignicons6-webfont-charmap.json'),
-            ('ph', 'phosphor.ttf', 'phosphor-charmap.json'),
-            ('ri', 'remixicon.ttf', 'remixicon-charmap.json'),
-            ('msc', 'codicon.ttf', 'codicon-charmap.json'),
-        )
+        # Verify that vendorized fonts are not corrupt
+        for fargs in BUNDLED_FONTS:
+            ttf_filename = fargs[1]
+            ttf_hash = MD5_HASHES.get(ttf_filename, None)
+            if ttf_hash is not None:
+                hasher = hashlib.md5()
+                with open(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                       "fonts", ttf_filename), 'rb') as f:
+                    content = f.read()
+                    hasher.update(content)
+                ttf_calculated_hash_code = hasher.hexdigest()
+                if ttf_calculated_hash_code != ttf_hash:
+                    raise FontError(u"Font is corrupt at: '{0}'".format(
+                                    os.path.join(directory, ttf_filename)))
```

Comment:
I think `os.path.join(directory, ttf_filename)` needs to be replaced with `os.path.join(os.path.dirname(os.path.realpath(__file__)), "fonts", ttf_filename)`

Reply:
Fixed at 3faefb25886b536adcbaf832804f38bab1a9264a